src/gwt/src/org/rstudio/studio/client/common/spelling/TypoSpellChecker.java

```diff
@@ -372,7 +382,7 @@ private void loadDictionary()
    {
       final String language = userPrefs_.spellingDictionaryLanguage().getValue();
 
-      if (typoLoaded_ && loadedDict_.equals(language))
+      if (!userPrefs_.realTimeSpellchecking().getValue() || typoLoaded_ && loadedDict_.equals(language))
```

Comment:
Could you use parens to group this so it's clear that the intent is to group the first two tests? (presuming that's the intent -- this isn't clear to me at first glance)

Reply:
It is not the intent. We want to exit out of this function if either realtime is turned off or the dictionary is loaded AND it's the correct dictionary. I'll put it on another line so it's a bit more clear. This is actually just reverting it to the exact same code from before the prior PR to here.